api/pkg/apps/cms/server.go

```diff
@@ -38,13 +39,18 @@ func NewServer(ctx context.Context, o *server.GenericServerOptions) (*Server, er
 		return nil, err
 	}
 
+	var hydraAdmin admin.ClientService
+	if o.HydraAdminClient != nil {
+		hydraAdmin = o.HydraAdminClient.Admin
+	}
```

Comment:
Why whould this be nil ?

Reply:
No idea, I was running into this when running the tests so I added this.